deploy/basic/terraform/scripts/node.sh

```diff
@@ -34,24 +52,30 @@ MUSHOP_APP_URI=$(curl -L http://169.254.169.254/opc/v1/instance/metadata | jq -j
 WALLET_URI=$(curl -L http://169.254.169.254/opc/v1/instance/metadata | jq -j ".wallet_par")
```

Comment:
The `http://169.254.169.254/opc/v1/instance/metadata` should be a variable, as the usage repeats 5 times.

Reply:
Will raise as a separate PR, since that was not part of this change set